llvm/lib/Support/raw_ostream.cpp

```diff
@@ -893,7 +893,13 @@ void raw_fd_ostream::anchor() {}
 //  outs(), errs(), nulls()
 //===----------------------------------------------------------------------===//
 
+static thread_local raw_fd_ostream *OutsOverride = nullptr;
+static thread_local raw_fd_ostream *ErrsOverride = nullptr;
```

Comment:
These should probably use `LLVM_THREAD_LOCAL` so they are just normal globals when `LLVM_ENABLE_THREADS` is false.

Reply:
Ah, sure, done!